data/bash-completion/fwupdtool.in

```diff
@@ -64,22 +64,22 @@ _fwupdtool_opts=(
 _show_filters()
 {
 	local flags
-	flags="$(command @libexecdir@/fwupdtool get-device-flags 2>/dev/null)"
+	flags="$(command fwupdtool get-device-flags 2>/dev/null)"
 	COMPREPLY+=( $(compgen -W "${flags}" -- "$cur") )
 }
 
 _show_firmware_types()
 {
 	local firmware_types
-	firmware_types="$(command @libexecdir@/fwupdtool get-firmware-types 2>/dev/null)"
+	firmware_types="$(command fwupdtool get-firmware-types 2>/dev/null)"
 	COMPREPLY+=( $(compgen -W "${firmware_types}" -- "$cur") )
 }
 
 
 _show_plugins()
 {
 	local plugins
-	plugins="$(command @libexecdir@/fwupdtool get-plugins 2>/dev/null)"
+	plugins="$(command fwupdtool get-plugins 2>/dev/null | sed -e '/^[a-zA-Z]/!d' -e '/ /d' -e 's/:$//')"
```

Comment:
Doesn't need to be this PR, but a future change we should probably fix get-plugins to have a json output too to make this cleaner.

Reply:
I think I've completed the modifications except this one.